obs/installation-images.spec

```diff
@@ -93,14 +93,12 @@ ExclusiveArch:  do_not_build
 %endif
 
 %if "%flavor" == "SMO"
+%if 0%{?is_smo}
```

Comment:
There is no `%is_smo`.

Reply:
It will be defined in SUSE:SLE-15-SP3:Update:Products:MIcroOS51, which is the only project where I want this flavour to build.